src/components/signing_official_table/SigningOfficialTable.js

```diff
@@ -401,7 +401,15 @@ export default function SigningOfficialTable(props) {
             style: Object.assign({}, Styles.MEDIUM_DESCRIPTION, {
               fontSize: '16px',
             }),
-          },['My Institution\'s Researchers. Issue or Remove Researcher privileges below.']),
+          },[
+            'My Institution\'s Researchers. Issue or Remove Researcher privileges below. ',
+            a({
+              rel: 'noopener noreferrer',
+              href: 'https://broad-duos.zendesk.com/hc/en-us/articles/360060402751-Signing-Official-User-Guide',
+              target: '_blank',
+              id: 'so-console-info-link'},
+            ['Click Here for more information'])
+          ]),
```

Comment:
Nit: The spacing between the Search Bar and Add New Researcher Button has been lost due to the added text. Would it be possible to add some space back, possibly by putting the link on the following line? 

![Add New Researcher Before](https://user-images.githubusercontent.com/81024249/171934612-73c9bc8e-01d7-4dd5-b4bc-20c52810f98c.png)
![Add New Researcher After](https://user-images.githubusercontent.com/81024249/171934630-bda8b4ec-5842-4003-a87e-6d6e27b7fd0d.png)

Reply:
added a lil padding to this!